packages/forms-web-app/src/dynamic-forms/journey-response.js

```diff
@@ -26,7 +26,7 @@ class JourneyResponse {
 	 * creates an instance of a JourneyResponse
 	 * @param {JourneyType} journeyId
 	 * @param {string} referenceId
-	 * @param {Record<string, unknown> | undefined} answers
+	 * @param {Record<string, unknown> | null} answers
```

Comment:
`answers` was passed in as null but never `undefined`, guess it works either way but might as well use the right type